src/components/ebay-breadcrumb/examples/1-breadcrumb-prevent-default/template.marko

```diff
@@ -0,0 +1,7 @@
+
+<ebay-breadcrumb aria-label="You are here" prevent-default=true>
```

Comment:
For boolean attributes, you can emit the `=true` part, since we're just looking for presence.

Reply:
I think you mean "omit" the `=true` part, right @yomed?